conda/testing/gateways/helpers.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+
+import json
+import os
+from pathlib import Path
+from subprocess import run
+
+try:
+    run(["minio", "-v"], check=True)
+    have_minio = True
+    have_minio_error = None
+except Exception as exc:
+    have_minio = False
+    have_minio_error = exc
```

Comment:
It's worth using `conda.cli.find_executable` to make this cross-platform compatible (aka Windows ;).